### PR TITLE
Redesign "README" file

### PR DESCRIPTION
**Pull request by BySwiizen**
### What is the purpose of this pull request?

1. Redesign of the "README" file.

#### Changes

- [x] Use GitHub checklists to list things you've done or are still working on.

### CODE REVIEW

**Comment by datatags** [comment]
I'm still not sure about having the author be an assignee because it sends the author an unnecessary notification as soon as the PR is opened. Is there a benefit to having the author as the assignee? If not, I think it's better to just leave out the assignees.

**Comment by BySwiizen** [comment]
> I'm still not sure about having the author be an assignee because it sends the author an unnecessary notification as soon as the PR is opened. Is there a benefit to having the author as the assignee? If not, I think it's better to just leave out the assignees.

I find this useful because if a PR contains several contributors, we can know who the author is. If notifications are also configurable in GitHub preferences. We can possibly do a test and see if it is disturbing

**Comment by datatags** [comment]
Not sure how I ended up commenting on the wrong PR 😅 but anyway, when I did the test PR earlier it did send me an email, so that's why I mentioned it. Regarding the PR author, it's already clearly visible in a couple places in the UI, so I don't see the need for it to also be in the assignees
uc pr

**Comment by BySwiizen** [comment]
> Not sure how I ended up commenting on the wrong PR 😅 but anyway, when I did the test PR earlier it did send me an email, so that's why I mentioned it. Regarding the PR author, it's already clearly visible in a couple places in the UI (even if there were other contributors), so I don't see the need for it to also be in the assignees uc pr

It's as you want.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,66 @@
...
+[![Downloads](https://img.shields.io/spiget/downloads/117279?color=blue)][spigotmc-link]
+
 # UltraCosmetics GUI Addon
 
-Helper plugin for the [UC-GUI resource pack](https://github.com/dennishzg/uc-gui). Addon for the main plugin, [UltraCosmetics](https://github.com/datatags/UltraCosmetics).
+UltraCosmetics GUI Addon is a [UltraCosmetics](https://github.com/datatags/UltraCosmetics) addon for [UC-GUI resource pack](https://github.com/dennishzg/uc-gui)!
+
+---
+
+## ? Compatibility
```
Comment (datatags) [inline README.md:15]: Are these supposed to be emojis like in the main UC readme? They're not rendering in the preview for me

Comment (BySwiizen) [reply README.md]: Thanks Windows... Fixed!

**Review by datatags — APPROVED** [review]

**Pull request merged**